api/ManifestApp/views.py

```diff
@@ -10,4 +10,10 @@ def index(request):
 class NodeList(ListCreateAPIView):
 
     queryset = NodeData.objects.all().order_by("VSN")
```

Comment:
do you still need this?  now that you have the `get_queryset` ?

Reply:
no, I should delete this